front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -44,7 +43,17 @@ const ContentBuilderSettings = () => {
   });
 
   return selected && isEnabled && selected.id !== ROOT_NODE ? (
-    <StyledBox bgColor={colors.adminDarkBackground} p="20px" w="400px">
+    <StyledBox
+      id="settings-box"
+      position="fixed"
+      right="0"
+      top="156px"
```

Comment:
Can we make this not such a magical number? I think it's menuHeight *2, or am I mistaken?